arelle/Locale.py

```diff
@@ -37,10 +35,14 @@
     "sk": "SK", "sl": "SI", "sq": "AL", "sr": "RS", "sv": "SE", "th": "TH",
     "tr": "TR", "uk": "UA", "ur": "PK", "vi": "VN", "zh": "CN"}
 
-def getUserLocale(localeCode: str = '') -> tuple[LocaleDict, str | None]:
-    # get system localeconv and reset system back to default
+
+def getUserLocaleUnsafe(localeCode: str = '') -> tuple[LocaleDict, str | None]:
```

Comment:
Prefix this function with an underscore? I don't see any good reason why we'd want someone to call this from another module.